sources/views/data.js

```diff
@@ -0,0 +1,34 @@
+import {JetView} from "webix-jet";
+import Table from "views/table.js";
+import {countries} from "models/countries.js";
+import {statuses} from "models/statuses.js";
+
+
+export default class DataView extends JetView {
+	config() {
+		return {
+			rows: [
+				{
+					view: "segmented", value: "Countries", multiview: true, options: [
+						{value: "Countries", id: "Countries"},
+						{value: "Statuses", id: "Statuses"}
+					]
+				},
+				{
+					cells: [
+						{
+							id: "Countries", rows: [
```

Comment:
here you also should prefer  localIds, but for `multiview:true` magic you need "global" ones
you can avoid this by setting an onChange handler for segmented and  calling `this.$$(id).show` to switch cells

Reply:
one small thing left: now you don't need `multiview:true` setting for segmented button